### PR TITLE
Add v3.45.0 changelog

### PR DESCRIPTION
**Pull request by rileykarson**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,83 @@
-## 3.44.0 (Unreleased)
+## 3.45.0 (Unreleased)
+
+UNKNOWN CHANGELOG TYPE:
+* Add iam policy resources for netbook instance ([#2605](https://github.com/hashicorp/terraform-provider-google-beta/pull/2605))
+* Add taxonomy and policy_tag resources to data_catalog product ([#2626](https://github.com/hashicorp/terraform-provider-google-beta/pull/2626))
```
Comment (c2thorn) [inline CHANGELOG.md:5]: Should these replace the corresponding `New Resource` lines under the Features section? Otherwise we could just remove this block

Comment (rileykarson) [reply CHANGELOG.md]: Just forgot to delete these lines!

**Review by c2thorn — APPROVED** [review]

**Comment by ralbertazzi** [comment]
Eagerly waiting for it to be released 🙏 thanks guys!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,79 @@
...
+
+FEATURES:
+* **New Data Source:** `google_spanner_instance` ([#2602](https://github.com/hashicorp/terraform-provider-google-beta/pull/2602))
+* **New Resource:** `google_notebooks_instance_iam_binding` ([#2605](https://github.com/hashicorp/terraform-provider-google-beta/pull/2605))
+* **New Resource:** `google_notebooks_instance_iam_member` ([#2605](https://github.com/hashicorp/terraform-provider-google-beta/pull/2605))
+* **New Resource:** `google_notebooks_instance_iam_policy` ([#2605](https://github.com/hashicorp/terraform-provider-google-beta/pull/2605))
+* **New Resource:** `access_context_manager_access_level_condition` ([#2595](https://github.com/hashicorp/terraform-provider-google-beta/pull/2595))
+* **New Resource:** `google_bigquery_routine` ([#2622](https://github.com/hashicorp/terraform-provider-google-beta/pull/2622))
+* **New Resource:** `google_iam_workload_identity_pool` ([#2623](https://github.com/hashicorp/terraform-provider-google-beta/pull/2623))
+* **New Resource:** `google_notebooks_instance_iam_binding` ([#2605](https://github.com/hashicorp/terraform-provider-google-beta/pull/2605))
```
Comment (ralbertazzi) [inline CHANGELOG.md:18]: This is repeated (line 12)

Comment (rileykarson) [inline CHANGELOG.md:18]:
```suggestion
```

Comment (rileykarson) [reply CHANGELOG.md]: Thanks!

**Pull request merged**